src/store/store.ts

```diff
@@ -58,10 +59,19 @@ const baseStore = create<State & Actions>(set => {
   return {
     ...initialState,
     navigateTo: nextPage =>
-      set(state => ({
+      set(state => {
+        if (nextPage === "login" || nextPage === "sign-up") {
```

Comment:
the login page doesn't create a closed loop and I could reasonably go into 'login' from 'explore' and decide I just want to go back, so maybe this evaluation should look only at "sign-up" ?